src/html/hardware.html

```diff
@@ -65,7 +65,7 @@ <h1><b>ExpressLRS</b></h1>
 	<br>
 	<div class="mui-container-fluid mui-col-sm-10 mui-col-sm-offset-1">
 		<div id="custom_config" class="mui-panel" style="display:none; background-color: #FFC107;">
-			This hardware configuration has been customised. This can be safely ignored if this is a custom hardware build or for testing purposes.<br>
+			This hardware configuration has been customized. This can be safely ignored if this is a custom hardware build or for testing purposes.<br>
```

Comment:
We speak the Queens english here 😆

Reply:
Whoa, you learn something every day. Did you already teach me that and I forgot? Because I feel like we may have had this conversation before. 😓

Reply:
Nah, maybe it was my Aussie compatriots?